cms/apps/media/models.py

```diff
@@ -91,6 +91,19 @@ class File(models.Model):
         help_text="Text used for screen readers"
     )
 
+    # For images to zoom in on a certain point when cropping/scaling.
+    focal_x = models.CharField(
```

Comment:
It feels to me that this should be IntegerField or DecimalField, as it will be storing numbers from 0 to 100 - it should immediately throw an exception if you try to put an invalid value in here. (If for technical reasons it does need to be a CharField, 20 seems like a rather large `max_length`.)